kedro-datasets/setup.py

```diff
@@ -0,0 +1,41 @@
+import re
+from codecs import open
+from os import path
+
+from setuptools import find_packages, setup
+
+name = "kedro-datasets"
+here = path.abspath(path.dirname(__file__))
+
+# get package version
+package_name = name.replace("-", "_")
+with open(path.join(here, package_name, "__init__.py"), encoding="utf-8") as f:
+    version = re.search(r'__version__ = ["\']([^"\']+)', f.read()).group(1)
+
+# get the dependencies and installs
+with open("requirements.txt", "r", encoding="utf-8") as f:
+    requires = [x.strip() for x in f if x.strip()]
+
+# get test dependencies and installs
+with open("test_requirements.txt", "r", encoding="utf-8") as f:
+    test_requires = [x.strip() for x in f if x.strip() and not x.startswith("-r")]
+
+
+# Get the long description from the README file
+with open(path.join(here, "README.md"), encoding="utf-8") as f:
+    readme = f.read()
+
+setup(
+    name=name,
+    version=version,
+    description="Kedro-Datasets is the core plugin that power Kedro's DataCatalog",
```

Comment:
I think we should change this also. Maybe this?
```suggestion
    description="Kedro-Datasets is where you can find all of Kedro's data connectors.",
```